docs/data_management/res/individual_allocation_functional_roles.csv

```diff
@@ -0,0 +1,6 @@
+﻿ Permissions/Responsibilities,Alloacted user,Other user
+Responsible for allocation,<span>&#x2714;</span>,
```

Comment:
In the Globus Groups PR I used the following literal text. It is the word Yes followed by a space character and then the emoji checkmark character at codepoint U+2714.

```
Yes ✔️
```

We want to use both the word Yes and the checkmark for accessibility reasons.

CSVs with UTF-8 encoding are accepted by our rendering stack, so you can use checkmarks directly, for example. So no need to use embedded html. It will be easier to maintain the literal checkmark character.